.github/CODEOWNERS

```diff
@@ -1678,6 +1678,8 @@ x-pack/test/security_solution_cypress/cypress/tasks/expandable_flyout  @elastic/
 /x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine @elastic/security-detection-engine
 /x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/rules/rule_gaps.ts @elastic/security-detection-engine
 /x-pack/test/security_solution_api_integration/test_suites/lists_and_exception_lists @elastic/security-detection-engine
+/x-pack/test/functional/es_archives/asset_criticality @elastic/security-detection-engine
+/x-pack/test/functional/es_archives/security_solution @elastic/security-detection-engine
```

Comment:
I don't think that /x-pack/test/functional/es_archives/security_solution - all archives should be owned as detection engine team

Reply:
ah ok, so then it's divided from there right?

Reply:
there a lot of archives inside this folder that can be used by several teams.

I think's its ok for this PR to only keep asset_criticality for DE

Reply:
Ok, makes sense.  I feel a kibana-wide email coming on then 😉 

I'll keep the pr open for now, but back in draft.  

First I'll look around some more for your team specifically.
Then I get the feeling that email will help a ton.